commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java

```diff
@@ -174,8 +174,7 @@ public CPDefinitionInventory updateCPDefinitionInventory(
 			String lowStockActivity, boolean displayAvailability,
 			boolean displayStockQuantity, int minStockQuantity,
 			boolean backOrders, int minOrderQuantity, int maxOrderQuantity,
-			String allowedOrderQuantities, int multipleOrderQuantity,
-			ServiceContext serviceContext)
```

Comment:
@igorbeslic To avoid major version change, can we add a new method, deprecate the current one and call the new method from the old one?  
We will delete those deprecate methods with next major Commerce Version.
As we do have customer online we have to be careful.